digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/repository/DefaultSyncLogRepository.java

```diff
@@ -0,0 +1,41 @@
+package org.digit.health.sync.repository;
+
+import org.digit.health.sync.web.models.dao.SyncLogData;
+import org.digit.health.sync.web.models.request.SyncLogSearchDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Repository("defaultSyncLogRepository")
+public class DefaultSyncLogRepository implements SyncLogRepository{
+
+    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+
+    private final SyncLogQueryBuilder syncLogQueryBuilder;
+
+    @Autowired
```

Comment:
Line length should not be more than 100-120 chars

Reply:
Updated